requirements.txt

```diff
@@ -0,0 +1,4 @@
+autopep8==2.0.2
+numpy==1.24.3
+pycodestyle==2.10.0
+tomli==2.0.1

```

Comment:
Is there a reason to pin packages version?

Reply:
It can help with debugging issues since there's a pinned version to check against, and all devs can be assumed to be using the same version.